qiskit/qpy/interface.py

```diff
@@ -93,6 +93,7 @@ def dump(
     use_symengine: bool = False,
     version: int = common.QPY_VERSION,
     annotation_factories: Optional[Mapping[str, Callable[[], annotation.QPYSerializer]]] = None,
+    use_rust: bool = True,
```

Comment:
As we discussed offline with the team, the Rust path for `dump` and `load` should be the only path and we don't want to use feature flags for Rust-QPY. So putting this comment here as a reminder to remove those before merging.